GitUI/Translation/English.xlf

```diff
@@ -1337,7 +1337,7 @@ Please make sure git (Git for Windows or cygwin) is installed or set the correct
         <target />
       </trans-unit>
       <trans-unit id="cherrypickSelectedLinesToolStripMenuItem.Text">
-        <source>Cherry pick selected lines</source>
+        <source>Stage selected line(s)</source>
```

Comment:
The translation app should not add these strings to the .xlf (as the strings are translated in Strings.cs).
Similar changes in TranslationStrings are handled correctly